torch/csrc/lazy/core/ir_builder.h

```diff
@@ -73,59 +73,6 @@ struct IrBuilder {
       const size_t& num_outputs = 1,
       const hash_t& hash_seed = static_cast<uint32_t>(0x5a2d296e9)) const = 0;
 
-  // View op nodes
```

Comment:
Oops, looks like we still need all these for XLA.

Reply:
Wait a minute, XLA's implementations just place holders... Let me make a companion PR.